web/ui/.gitignore

```diff
@@ -0,0 +1,34 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```

Comment:
why we need ignore yarn-debug.log/yarn-error.log?

Reply:
1.This is automatically generated at the time of creation
2.The front-end local installation depends on the error, the generated debug log file should not be committed